conf/application.conf

```diff
@@ -294,3 +294,10 @@ bootstrap.http.headersAllowlist = [ "Gov-Test-Scenario", "Location", "X-Request-
 internalServiceHostPatterns = [ "localhost" ]
 
 accessibility-statement.service-path = "/income-tax-view-change"
+
+###################### DECIDE WHAT TO SET ENCRYPTION KEY TO ###############################
+useEncryption = false
```

Comment:
it would make sense to put these together with the key, like:

```
encryption {
  useEncryption = false
  key = "QmFyMTIzNDVCYXIxMjM0NQ=="
}
```